config/locales/ca.yml

```
@@ -43,47 +74,69 @@ ca:
       content: "Encara no hi ha cap %{resource_name}."
       link: "Crea'n un/a"
     batch_actions:
-      button_label: "les accions per lots"
-      default_confirmation: "¿Esteu segur que voleu fer-ho?"
-      delete_confirmation: "¿Està segur que desitja eliminar aquests %{plural_model}?"
+      button_label: "Accions per lots"
+      default_confirmation: "Segur que voleu fer-ho?"
+      delete_confirmation: "Segurs que voleu eliminar aquests %{plural_model}?"
       succesfully_destroyed:
-        one: "Va destruir amb èxit 1 %{model}"
-        other: "Va destruir amb èxit %{count} %{plural_model}"
-      selection_toggle_explanation: "(Selecció de Canviar)"
+        one: "1 %{model} eliminat"
+        other: "%{count} %{plural_model} eliminats"
+      selection_toggle_explanation: "(Invertir la selecció)"
       action_label: "%{title} seleccionat"
       labels:
-        destroy: "esborrar"
+        destroy: "Esborrar"
     comments:
-      body: "Cos"
-      author: "autor"
+      created_at: "Creat el"
+      resource_type: "Tipus de recurs"
+      author_type: "Tipus d'author"
+      body: "Missatge"
+      author: "Autor"
       add: "Afegeix comentari"
+      delete: "Elimina comentari"
+      delete_confirmation: "Esteu segurs que voleu eliminar aquest comentari?"
       resource: "Recurs"
-      no_comments_yet: "No hi ha comentaris"
-      title_content: "comentaris (%{count})"
+      no_comments_yet: "Sense comentaris"
+      author_missing: "Anònim"
+      title_content: "Tots els comentaris (%{count})"
       errors:
-        empty_text: "El comentari no es va salvar, el text estava buida."
+        empty_text: "El comentari no s'ha desat, no hi havia text."
     devise:
+      username:
+        title: "Usuari"
+      email:
+        title: "Email"
+      subdomain:
+        title: "Subdomini"
+      password:
+        title: "Contrasenya"
+      password_confirmation:
+        title: "Confirmeu la contrasenya"
+      sign_up:
+        title: "Doneu-vos d'alta"
+        submit: "Doneu-vos d'alta"
       login:
-        title: "iniciar sessió"
-        remember_me: "Recordar"
-        submit: "iniciar sessió"
+        title: "Identifiqueu-vos"
+        remember_me: "Recorda'm"
+        submit: "Identifiqueu-vos"
       reset_password:
-        title: "Heu perdut la contrasenya?"
+        title: "Heu oblidat la contrasenya?"
         submit: "Restablir la contrasenya"
       change_password:
         title: "Canvieu la contrasenya"
-        submit: "Canviar la contrasenya"
+        submit: "Canvia'm la contrasenya"
       unlock:
         title: "Reenvia instruccions per a desbloquejar"
         submit: "Reenvia instruccions per a desbloquejar"
       resend_confirmation_instructions:
         title: "Reenviar instruccions de confirmació"
         submit: "Reenviar instruccions de confirmació"
       links:
-        sign_in: "Registrar"
+        sign_up: "Sign up"
+        sign_in: "Sign in"
```

These were added but without translation. I used the `title` values from the `sign_up` and `login` keys.

```suggestion
        sign_up: "Doneu-vos d'alta"
        sign_in: "Identifiqueu-vos"
```